fastlane/lib/fastlane/actions/badge.rb

```diff
@@ -16,7 +16,11 @@ def self.run(params)
           shield_gravity: params[:shield_gravity],
           shield_no_resize: params[:shield_no_resize]
         }
-        Badge::Runner.new.run(params[:path], options)
+        begin
+          Badge::Runner.new.run(params[:path], options)
+        rescue => e
+          UI.user_error!("Something went wrong while running badge: #{e}")
```

Comment:
Okay, understood. However eating the exception will make it difficult to debug the problem. Do you think we can also print out the stack trace here? We've needed this a few times now, maybe it's time to have a `Helper` method that prints the original error + stack trace? In `fastlane init` we should start using that as well 👍